contrib/depend/install.sh

```diff
@@ -126,6 +126,18 @@ xtensor_python() {
 
 }
 
+ubuntu_1804(){
+
+  sudo apt-get -qy install \
```

Comment:
Your script should not invoke sudo.  Users should be allowed to opt in sudo when invoking the script from outside, like `sudo your_script.sh`.